src/objects/Allocation__c.object

```diff
@@ -86,12 +86,23 @@
         <label>General Accounting Unit</label>
         <lookupFilter>
             <active>true</active>
+            <booleanFilter>(1 AND 2) OR 3</booleanFilter>
```

Comment:
@KietTruongSFDO - My only question on this was why was it necessary to update the Allocation object as part of this?  Is this needed?  If so, why?

Reply:
Actually, I get it...this was there to block it.  I need to review this with Pedro...just making it work with Payments, only partly fixes it.  If we cannot update the Opportunity Allocations, it will not work when it tries to finalize it.

Reply:
@pedrodalcol and I discussed this item.  We will need to do further work on this when he returns, since we will error when attempting to update the opportunity allocations for payment allocations changes (when there is an inactive GAU present), but we agreed that was a) not a reason to hold this up, and b) erroring for now, while we figure out what we really want to have happen is probably ok.